src/test/java/duke/ParserTest.java

```diff
@@ -1,22 +1,23 @@
 package duke;
 
-import duke.command.Command;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import duke.command.Command;
 
 public class ParserTest {
     @Test
-    public void dummyTest(){
+    public void dummyTest() {
```

Comment:
I think using another name for test method would be more appropriate. Perhaps using the test name convention `whatIsBeingTested_descriptionOfTestInputs_expectedOutcome`